docs/components/segmented-control.md

```diff
@@ -0,0 +1,102 @@
+# Segmented Control
+
+**KSegmentedControl** is used a like radio button group and is meant to toggle between mutually exclusive options.
+
+<KToggle v-slot="{ isToggled, toggle }">
+  <KSegmentedControl
+      :options="['On','Off']"
+      :selected="isToggled?'On':'Off'"
+      @toggled="toggle"
+  />
+</KToggle>
```

Comment:
using ktoggle switches the values even when you click the active button. Maybe do this:

```suggestion
<KToggle v-slot="{ isToggled, toggle }">
  <KSegmentedControl
      :options="['true','false']"
      :selected="`${isToggled}`"
      @toggled="state => `${isToggled}` !== state && toggle()"
  />
</KToggle>
```